headers/linux/if_xdp.h

```diff
@@ -70,6 +70,15 @@ struct xdp_umem_reg {
 	__u32 chunk_size;
 	__u32 headroom;
 	__u32 flags;
+	__u32 tx_metadata_len;
+};
+
+struct xdp_umem_reg_v1 {
+	__u64 addr; /* Start of packet data area */
+	__u64 len; /* Length of packet data area */
+	__u32 chunk_size;
+	__u32 headroom;
+	__u32 flags;
```

Comment:
This header file if copied directly from the kernel headers, so let's not add custom definitions to it; might as well have this one directly in the .c file. Or just use offsetof() instead...